tests/php/Core/EnvironmentTest.php

```diff
@@ -90,6 +90,16 @@ public function provideHasEnv()
         $scenarios = [];
         foreach ($setAsOptions as $setAs) {
             foreach ($valueOptions as $value) {
+                // A note about null:
+                // Setting `MY_VAR=` in a .env file currently should result in a value of `null` being
+                // returned by Environment::getEnv('MY_VAR'), though it's a weird behaviour and it really should
+                // return a blank string instead due to getenv() being string only
+                // Having null values in $_ENV, $_SERVER isn't supported by Environment getEnv() or hasEnv() due to
+                // a strange issue apparently to do with too many array method calls
+                // Setting null in putenv() is non-sensical due to it being string only
+                if ($value === null && $setAsOptions !== '.env') {
```

Comment:
```suggestion
                if ($value === null && $setAs !== '.env') {
```

Reply:
But instead of this, just remove `null` from the `$valueOptions` array, and _after the loop_ add the one valid `null` option:
```php
// `null` isn't a supported value outside of using the `.env` option.
$scenarios[] = [
    'setAs' => '.env,
    'value' => null,
    'expected' => true,
];
```
It requires way less explanation and it's a lot clearer what's going on.

Reply:
Updated